Standards

```diff
@@ -99,7 +99,9 @@ EXPLANATION: Customers should get back every part of their machine that they bro
 
 Conduct:
 
-23) Do not be drunk or high at work. 
+23) Do not be drunk or high at work.
+
+EXPLANATION: People loose their senses, and can't control body with responsibility. Brain will lost connection to real world and bad things always happen.
```

Comment:
"Brain will lose connection to the real world and bad things are going to happen."

Without changing the wording too much, I think that this would be a better way to write that sentence.